### ISSUE TITLE
Where can I download the pretained model based on Imagenet?

### ISSUE THREAD

**Issue by Jasonbaby** [titu1994/Keras-ResNeXt#9]
Hello, the ResNeXtImageNet() need some .h5 files, and where should these files be download?

**Comment by titu1994** [comment]
ResNeXt is not supported by Keras or Tensorflow. Therefore there are no weights.

**Comment by John1231983** [comment]
Hi. Because keras does not support group convolution yet. So how did you solve with it in resnext? Does it loss some performance without using it?

**Comment by titu1994** [comment]
I used lambda layers to slice off filters and word on them separately and then concatenated that. However, this is a low and mostly wrong implementation with none of the speed benefits of grouped convolutions.

**Comment by John1231983** [comment]
I see. So using your way just makes the code run slowly, and it will not affect the accuracy. Am I right?

**Comment by titu1994** [comment]
Theoretically, it may affect accuracy as well. And it's a lot slower, so I recommend not to use it. 

There are better models out there such as Dense and SparseNets, SENets and NASNet.

**Comment by John1231983** [comment]
SENets also use ResneXt :(. I am finding the Se-resneXT implementation

**Comment by titu1994** [comment]
The squeeze and excitation mechanism is generic to any architecture. There are se variants to ordinary ResNets, DenseNets, Inception and other types of nets in my repo. 

Coincidentally,vi would not advice to use the SE-ResNext in my repo either
